src/media/js/settings_app.js

```diff
@@ -19,6 +19,9 @@ define('settings_app',
         return window.location.search.indexOf('cache=false') === -1;
     }
 
+    var is_packaged_app = window.location.protocol === 'app:';
```

Comment:
Yeah. `capabilities` feels wrong too, as it's not really a capability. And at the same time I do want something I can overwrite for tests and something DRY.